include/glow/Runtime/HostManager/HostManager.h

```diff
@@ -213,6 +213,19 @@ class HostManager final {
 
   ~HostManager();
 };
+
+/// Helper to get the parameters in DeviceConfig from \p str. The \p str has
+/// multiple lines, and each line with this format : "str1" : "str2".
+llvm::StringMap<std::string> getBackendParams(std::string &str);
```

Comment:
Is it ever used independently of `generateDeviceConfigs` or is it just an implementation detail? If so, move the function definition as a `static` function into a cpp file where `generateDeviceConfigs` is defined.